jni/src/Control/Concurrent/PThread.hsc

```diff
@@ -0,0 +1,73 @@
+-- | PThread bindings
```

Comment:
Perhaps this should be renamed to `Foreign/PThread.hsc` or `Foreign.Concurrent.PThread.hsc`.